bin/oxd

```diff
@@ -155,6 +155,8 @@ def main():
 
     setup_logging(options)
 
+    continue_running = True
```

Comment:
Yeah it was still screwed up. The SIGTERM handler is supposed to handle it, but it must have never worked right.